VERSION

```diff
@@ -0,0 +1 @@
+3.0.0
```

Comment:
Could / should we use a way to automatically generate the version file (e.g. ```go generate```)?

Reply:
How would the version be determined?

Reply:
I'm not entirely sure, some options:
* Use the commit hash
* Use the latest tag (shaky)
* Use something like build date

Reply:
Although to be honest, I don't really have a good solution at the moment